answers.txt

```diff
@@ -1,4 +1,16 @@
 What is a gitignore file and what should it include for a Java project?
+
+The role of a .gitignore file is to tell git which files it should ignore before we make a commit. For JAVA files it should include compiled class file(*.class), log file (*.log), BlueJ files(*.ctxt), Mobile tool for Java files(.mtj.tmp/), Package files(*.jar, *.war, *.nar, *.ear ,*.zip , *.tar.gz, *.rar) and virtual machine crash logs(hs_err_pid*).
+
 What does fast-forward in Git mean?
+
```

Comment:
One thing to add is that, fast-forward merge happens only if master doesnt have any separate commits  i.e, it is not detached.

Reply:
Added the new point, committed the changes and created a new pull request.